game/scripts/npc/units/siltbreaker/npc_dota_creature_temple_guardian.txt

```diff
@@ -63,7 +63,7 @@
 
 		// Status
 		//----------------------------------------------------------------
-		"StatusHealth"				  "25000"
+		"StatusHealth"				  "12500"
```

Comment:
This is wayyyyy too much of a nerf. The two combined are squishier than tier 3's

Reply:
tier 3 health:
ogre 7500
charger 3000